doc/api/vm.md

```diff
@@ -187,10 +184,12 @@ Using the `timeout` or `breakOnSigint` options will result in new event loops
 and corresponding threads being started, which have a non-zero performance
 overhead.
 
-### script.runInNewContext(\[sandbox\[, options\]\])
+### script.runInNewContext(\[contextObject\[, options\]\])
 <!-- YAML
 added: v0.3.1
 changes:
+  - version: REPLACEME
```

Comment:
is this really needed? it's a positional argument so the name doesn't really matter.

Reply:
Yeah I don't think we need to add to the changes list for something like this.

Reply:
I did it for `vm.createContext()` because otherwise existing YAML matter for that method that mentions `sandbox` doesn't make any sense anymore. And once I did it for `vm.createContext()`, it seemed to make sense to add it for the other methods (like this one). 

But that's just an explanation of why I added it. I can remove it if that's not a convincing reason to leave it in.

Reply:
I was speaking more generally and not just this particular function or even this module. I don't think we should be documenting changes to parameter names.

Reply:
OK, I removed the YAML additions and fixed the `vm.createContext()` YAML by changing _sandbox_ to _first argument_.